<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Postulante</class>
 <widget class="QMainWindow" name="Postulante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>DNI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDni">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbEdad">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Sexo</string>
    </property>
    <widget class="QRadioButton" name="rbMasculino">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Masculino</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbFemenino">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Femenino</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Modalidad</string>
    </property>
    <widget class="QComboBox" name="cboModalidad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Modalidad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Practicante</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medio Tiempo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tiempo Completo</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Area</string>
    </property>
    <widget class="QComboBox" name="cboArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Desarrollo de Sistemas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Redes Informaticas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Soporte Tecnico</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAceptar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RESULTADOS</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>gratis-png-chico-usando-computadora-programador-programacion-de-computadora-software-de-computadora-programador.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
